tests/model_revision_4/test_hazard_models.py

```diff
@@ -101,3 +122,45 @@ def test_HazardRealizationCurve_table_save_get(self, adapted_model, generate_rev
         # assert res.sources_key() == 'c9d8be924ee7'
         # assert res.rlz == m.rlz TODO: need string coercion for sqladapter!
         # assert 0
+
+    def test_HazardAggregation_table_save_get(self, adapted_model, generate_rev4_aggregation_models):
+
+        m = next(generate_rev4_aggregation_models())
+        print(m)
+        mHAG = adapted_model.HazardAggregateCurve
+        m.save()
+        res = next(
+            mHAG.query(
+                m.partition_key,
+                mHAG.sort_key == m.sort_key,
+                # (mHRC.compatible_calc_fk == m.compatible_calc_fk)
+                # & (mHRC.producer_config_fk == m.producer_config_fk)
+                # & (mHRC.vs30 == m.vs30),  # filter_condition
+            )
+        )
+
+        print(res)
+        assert res.created.timestamp() == int(m.created.timestamp())  # approx
+        assert res.vs30 == m.vs30
+        assert res.imt == m.imt
+        assert res.values[0] == m.values[0]
+        assert res.sort_key == '-38.160~178.247:0250:PGA:mean:NSHM_DUMMY_MODEL'
+
+    def test_HazardAggregation_roundtrip_dataset(self, generate_rev4_aggregation_models, tmp_path):
```

Comment:
@chrisdicaprio this test illustrates saving the new HazardAggregateCurve models to dataset.